.travis.yml

```
@@ -30,25 +32,25 @@ install:
       export OS="Linux";
     fi
 
-    wget https://repo.continuum.io/miniconda/Miniconda3-latest-${OS}-x86_64.sh -O miniconda.sh
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-${OS}-x86_64.sh -O miniconda.sh
   - bash miniconda.sh -u -b -p $HOME/miniconda
   - source "$HOME/miniconda/etc/profile.d/conda.sh"
   - hash -r
 
   # Configure conda testing environment
   - conda config --set always_yes yes --set changeps1 no
-  - conda config --append channels conda-forge
+  - conda config --add channels conda-forge
   - conda update -q conda
 
   - conda create -q -n test-environment
-        python=$PYTHON_VERSION
-        fftw>=3.3.8
-        liblapack>=3.8
+        "python=$PYTHON_VERSION"
+        numpy
+        mkl
```

Following make numpy use mkl.
```suggestion
        "liblapack=*=*mkl"
```